cmd/postgres.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/postgresConfig/get"
+	"github.com/supabase/cli/internal/postgresConfig/update"
+	"github.com/supabase/cli/internal/utils/flags"
+)
+
+var (
+	postgresCmd = &cobra.Command{
+		GroupID: groupManagementAPI,
+		Use:     "postgres",
+		Short:   "Manage Postgres database",
+	}
+	postgresConfigCmd = &cobra.Command{
+		Use:   "config",
+		Short: "Manage Postgres database config",
+	}
+
+	postgresConfigGetCmd = &cobra.Command{
+		Use:   "get",
+		Short: "Update network restrictions",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return get.Run(cmd.Context(), flags.ProjectRef, afero.NewOsFs())
+		},
+	}
+
+	postgresConfigUpdateCmd = &cobra.Command{
+		Use:   "update",
+		Short: "Update Postgres database config.",
+		Long: `Overriding the default Postgres config could result in unstable database behaviour.
+Custom configuration also overrides the optimizations generated based on the compute add-ons in use.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return update.Run(cmd.Context(), flags.ProjectRef, postgresConfigValues, postgresConfigUpdateReplaceMode, afero.NewOsFs())
+		},
+	}
+
+	postgresConfigValues            []string
+	postgresConfigUpdateReplaceMode bool
+)
+
+func init() {
+	postgresCmd.PersistentFlags().StringVar(&flags.ProjectRef, "project-ref", "", "Project ref of the Supabase project.")
+	postgresCmd.AddCommand(postgresConfigCmd)
+	postgresConfigCmd.AddCommand(postgresConfigGetCmd)
+	postgresConfigCmd.AddCommand(postgresConfigUpdateCmd)
+
+	postgresConfigUpdateCmd.Flags().StringSliceVar(&postgresConfigValues, "config", []string{}, "Config overrides specified as a 'key:value' pair")
```

Comment:
```suggestion
	postgresConfigUpdateCmd.Flags().StringSliceVar(&postgresConfigValues, "config", []string{}, "Config overrides specified as a 'key=value' pair")
```

Can we use `=` to avoid confusion with other commands? For eg. `supabase inspect db:cache-hit` command already uses colon as a namespace separator.

Reply:
> db:cache-hit

I thought we wanted to change that? https://github.com/supabase/cli/pull/1070#issuecomment-1582623850

Anyway yeah, we use `foo=bar` for setting secrets